src/key.js

```diff
@@ -1320,6 +1351,7 @@ export function getPreferredSymAlgo(keys) {
     });
   });
   var prefAlgo = {prio: 0, algo: config.encryption_cipher};
+  // FIXME
```

Comment:
What needs to be fixed?

Reply:
The only thing that comes to mind is this: https://eslint.org/docs/rules/guard-for-in. I'll remove the FIXME.